static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -50,11 +63,41 @@ const hideCancelContribution = (): void => {
     $(CANCEL_CONTRIBUTION).addClass(IS_HIDDEN_CLASSNAME);
 };
 
-const hideContributionInfo = (): void => {
-    $(CONTRIBUTION_INFO).addClass(IS_HIDDEN_CLASSNAME);
+const displayCancelContributionForm = (): void => {
+    $(CANCEL_CONTRIBUTION_FORM).removeClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContributionForm = (): void => {
+    $(CANCEL_CONTRIBUTION_FORM).addClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContributionSuccessMessage = () => {
+    $(CANCEL_CONTRIBUTION_MSG_SUCCESS).addClass(IS_HIDDEN_CLASSNAME);
+};
+
+const displayCancelContributionLink = () => {
+    $(CANCEL_CONTRIBUTION_LINK).removeClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContributionLink = () => {
+    $(CANCEL_CONTRIBUTION_LINK).addClass(IS_HIDDEN_CLASSNAME);
+};
+
+const disableCancellationSubmit = () => {
+    $(CANCEL_CONTRIBUTION_SUBMIT).addClass(IS_DISABLE_CLASSNAME);
+    $(CANCEL_CONTRIBUTION_SUBMIT).attr('disabled', 'disabled');
```

Comment:
Setting the `disabled` property directly would be preferred over setting this attribute
```js
$(CANCEL_CONTRIBUTION_SUBMIT)[0].disabled = true;
```